omnipaxos_core/src/util.rs

```diff
@@ -147,6 +147,10 @@ where
         self.accepted_stopsign[Self::pid_to_idx(from)] = true;
     }
 
+    pub fn get_accepted_stopsign(&mut self, from: u64) -> bool {
```

Comment:
For functions that return `bool`, a good convention is to name them as questions that can be answered with true/false :D

```suggestion
    pub fn follower_has_accepted_stopsign(&mut self, pid: NodeId) -> bool {
```